redisinsight/api/src/main.ts

```diff
@@ -30,6 +30,11 @@ export default async function bootstrap(apiPort?: number): Promise<IApp> {
   }
 
   const { port, host } = serverConfig;
+
+  if (apiPort && apiPort !== port) {
```

Comment:
Let's be as simple as possible. Not sure that we must add additional check `apiPort !== port` here to avoid misunderstandings.